src/perf/lib/quicmain.cpp

```diff
@@ -37,18 +37,21 @@ QUIC_STATUS
 QuicMainStart(
     _In_ int argc,
     _In_reads_(argc) _Null_terminated_ char* argv[],
-    _In_ QUIC_EVENT StopEvent,
+    _In_ QUIC_EVENT* StopEvent,
     _In_ PerfSelfSignedConfiguration* SelfSignedConfig
     ) {
-    const char* TestName = GetValue(argc, argv, "TestName");
-    if (!TestName) {
-        WriteOutput("Must have a TestName specified. Ex: -TestName:Throughput\n");
+    if (argc < 3 || IsArg(argv[1], "?") || IsArg(argv[1], "help") || !IsArg(argv[2], "TestName")) {
         PrintHelp();
         return QUIC_STATUS_INVALID_PARAMETER;
     }
 
+    const char* TestName = GetValue(argc, argv, "TestName");
```

Comment:
Won't this collide with https://github.com/microsoft/msquic/pull/691/files#diff-a3358465a6163ede94a3c2edac7ee7bcR54

Since this will make argv[0] not be the executable name?

Reply:
Whoops, meant this to be on the line below.

Reply:
No, the goal here is to support the following:
```
quicping -help
```
and
```
quicping -TestName:Throughput -help
```
They each hit different `PrintHelp()` functions.

Reply:
Ok. Then my other comment will definitely need to be fixed then. There were 2 places it happened too.